modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/datatypes/tests/BaseIndexDataTypeTest.java

```diff
@@ -71,11 +73,13 @@ public void testNotEqLookUp() {
         T value3 = values.get(2);
 
         checkQuery("SELECT * FROM t WHERE test_key != $0 ORDER BY id")
+                .matches(containsTableScan("PUBLIC", "T"))
```

Comment:
seems u need to add this check into upper test too ?

Reply:
There is no need to do that as tests that extend this one have no tests (1)
And if they do they should add those checks case by case basis (2)

Reply:
I think additional tests are required here (with sorted index)
Seems we need additional table or add\remove `create index t_test_key_sorted_idx on t using tree(test_key)` appropriate idx on each test. WDYT ?